nilearn/plotting/find_cuts.py

```diff
@@ -209,6 +210,13 @@ def find_cut_slices(img, direction='z', n_cuts=12, spacing='auto'):
 
     data = _smooth_array(data, affine, fwhm='fast')
 
+    if (isinstance(n_cuts, numbers.Number) and n_cuts > 0):
```

Comment:
I am a bit unclear, at the moment I have done this.
```python
if not (isinstance(n_cuts, int) and n_cuts > 0):
        message = ("The number of cuts in the direction '%s' must be "
                   "an 'integer' between 0 and %d. You provided n_cuts=%s " % (
                       direction, this_shape, n_cuts))
        raise ValueError(message)
```
Please let me know if this is not the way it should be.